### DISCUSSION TITLE
Minify/Mangle/Uglify CSS Classnames in Next.js

### DISCUSSION THREAD

**Discussion by anonymouscatcher** [tailwindlabs/tailwindcss#9771] [Help]
Hi there,

Does anyone know how to Uglify tailwind class names in next js project? 

Thanks in advance

**Comment by driss-chelouati** [comment]
I think there is a webpack plugin for that, 
https://github.com/sndyuk/mangle-css-class-webpack-plugin
I wish there was a solution for Vite based projects.
